src/instant/instantTransactions.ts

```diff
@@ -47,14 +49,29 @@ export function tradePayWithETH(
     calls.tradePayWithETHNoProxy({ ...state, ...gasData } as InstantOrderData);
 
   return tx$.pipe(
-    switchMap((txState: TxState) =>
-      of(progressChange({
+    switchMap((txState: TxState) => {
+
+      if (txState.status === TxStatus.Success) {
+        return of(progressChange({
+          ...initialProgress,
+          tradeTxStatus: txState.status,
+          tradeTxHash: getTxHash(txState),
+          summary: {
+            soldToken: txState.meta.args.sellToken,
```

Comment:
Why summary contains soldToken, boughtToken if state already contains that sellToken, buyToken?